tests/storage_test.go

```diff
@@ -25,13 +25,15 @@ package tests
 import (
 	"fmt"
 	"net/http"
+	"strings"
 	"testing"
 	"time"
 
 	"github.com/skydive-project/skydive/storage/elasticsearch"
 	"github.com/skydive-project/skydive/topology/graph"
 )
 
+const indexPrefix = "test_skydive_mwnxf0f2"
```

Comment:
Maybe put the three following into a single const block, and remove redundent 'string' type specification.

Why is the 'mwnxf0f2' needed?